### ISSUE TITLE
Regarding fine-tune step for k_shot in Meta-cross work

### ISSUE THREAD

**Issue by Aniruddha-JU** [microsoft/vert-papers#13]
in your code for k_shot experiments,  which fine-step results should be reported? Either the last step or maximum among the steps?

**Comment by QianhuiWu** [comment]
> in your code for k_shot experiments, which fine-step results should be reported? Either the last step or maximum among the steps?

Hi, sorry for the late reply...
For all k-shot experiments, we take the k-shot examples to fine-tune the meta-trained model by 10 steps, then evaluate the fine-tuned model with the target-language test set.

**Comment by Aniruddha-JU** [comment]
Thanks for your reply, 
Can we also fine-tune the meta-trained model of a validation dataset with-shot example? then evaluate it on the target dataset on the highest validation value steps? is it also right approach?

**Comment by QianhuiWu** [comment]
> Thanks for your reply,
> Can we also fine-tune the meta-trained model of a validation dataset with-shot example? then evaluate it on the target dataset on the highest validation value steps? is it also right approach?

In my opinion, it's up to your experimental setting? (If there's a given valid set in your datasets, I think it's okay. However, if we consider the extreme low-resource setting, is it reasonable to extract a valid set from the low-resource data would be a question.
